test/production/app-dir/build-output-tree-view/build-output-tree-view.test.ts

```diff
@@ -14,37 +14,34 @@ describe('build-output-tree-view', () => {
     beforeAll(() => next.build())
 
     it('should show info about prerendered and dynamic routes in a tree view', async () => {
-      // TODO: Show cache info (revalidate/expire) for app router, and use the
-      // same for pages router instead of the ISR addendum.
-
       // TODO: Fix double-listing of the /ppr/[slug] fallback.
 
       expect(getTreeView(next.cliOutput)).toMatchInlineSnapshot(`
-       "Route (app)                             Size    First Load JS
-       ┌ ○ /_not-found                         N/A kB         N/A kB
-       ├ ƒ /api                                N/A kB         N/A kB
-       ├ ○ /api/force-static                   N/A kB         N/A kB
-       ├ ○ /app-static                         N/A kB         N/A kB
-       ├ ○ /cache-life                         N/A kB         N/A kB
-       ├ ƒ /dynamic                            N/A kB         N/A kB
-       ├ ◐ /ppr/[slug]                         N/A kB         N/A kB
-       ├   ├ /ppr/[slug]
-       ├   ├ /ppr/[slug]
-       ├   ├ /ppr/days
-       ├   └ /ppr/weeks
-       └ ○ /revalidate                         N/A kB         N/A kB
-       + First Load JS shared by all           N/A kB
+       "Route (app)                      Size  First Load JS  Revalidate  Expire
+       ┌ ○ /_not-found                N/A kB         N/A kB
```

Comment:
maybe just me, I feel the left aligned was intended, visually more comfortable, and the JS column as the previous last column, it's right aligned.

Horizontally wide feels hard to read. Having two new columns feel like a lot more information to view and match visually. I quite like the 1st iteration where they stay together. We can explain `"Revalidate / Expire"` in title.